backend/typescript/services/implementations/campService.ts

```diff
@@ -124,13 +210,22 @@ class CampService implements ICampService {
   async generateCampersCSV(campId: string): Promise<string> {
     try {
       const campers = await this.getCampersByCampId(campId);
+
       if (campers.length === 0) {
         // if there are no campers, we return an empty string
         return "";
       }
+
+      const flattenedCampers = campers.map((camper) => {
+        const { formResponses, ...formObj } = camper;
+        return {
+          ...formResponses,
+          ...formObj,
+        };
+      });
       // grabbing column names
-      const fields = Object.keys(campers[0]);
-      const csvString = await generateCSV({ data: campers, fields });
+      const fields = Object.keys(flattenedCampers[0]);
```

Comment:
Is it possible for campers to have different formResponses keys? Wondering if we need to take the union of keys for all campers.

Reply:
a camp has a specific set of form questions we are retrieving from, i.e. baseCamp has formQuestions field containing all the questions. By union, did you mean taking union of all campers from all the camps? cc: @annechung

Reply:
discussed a bit offline but no not a union of all questions from all camps, but from all campers i the same camp!

Reply:
oh whoops i thought i responded to this - it should be just the union of all campers from a specific campsession